test/integration-cmake-test.js

```diff
@@ -6,6 +6,9 @@ var rm = require('rimraf')
 
 var cwd = path.join(__dirname, 'native-module-cmake')
 
+if (process.arch !== 'ia32') { // see https://github.com/cmake-js/cmake-js/issues/186
```

Comment:
```suggestion
if (process.platform !== 'win32' || process.arch !== 'ia32') { // see https://github.com/cmake-js/cmake-js/issues/186
```

Doesn't actually make a difference, it's just to communicate intent.